server/grpc_service.go

```diff
@@ -102,7 +102,12 @@ func (s *GrpcServer) GetMembers(context.Context, *pdpb.GetMembersRequest) (*pdpb
 	members, err := s.Server.GetMembers()
 	if err != nil {
 		return &pdpb.GetMembersResponse{
-			Header: s.wrapErrorToHeader(pdpb.ErrorType_UNKNOWN, err.Error()),
+			Header: &pdpb.ResponseHeader{
+				Error: &pdpb.Error{
```

Comment:
Will the missing cluster ID field be a problem?

Reply:
I distinguish the two situations in the original `s.Server.GetMembers()`.